src/modules/list.marko

```diff
@@ -0,0 +1,314 @@
+<div id="list">
+    <section-header metadata=input.metadata name=input.name/>
+    <p>A list component is used to display certain types of data in a structured format. It can be used to display a list of items, a list of links, or a list of buttons.</p>
+
+
+    <h3 id="list-default">Default List</h3>
+   <div class="demo">
+        <div class="demo__inner">
+            <div class="list">
+                <div class="list__item">
+                    <span class="list__leading">
+                        <svg aria-hidden="true" class="icon icon--16" height="16" width="16">
+                            <icon-symbol name="folder-16"/>
+                        </svg>
+                    </span>
+                    <span class="list__body">
+                        Text 1
+                    </span>
+                </div>
+                <div class="list__item">
```

Comment:
1. The focus indicator looks to be cut off. It might need a pixel or two of margin? 
2. The switch looks to be disabled and does not get focus since there is a `tabindex="-1"` on it.

<kbd><img width="572" alt="image" src="https://github.com/user-attachments/assets/9f2a8c3c-0c7f-4957-b26c-1e3734f6bba4"></kbd>